.gitignore

```diff
@@ -15,3 +15,5 @@
 /.release
 .elixir.plt
 erl_crash.dump
+/lib/*/cover
+/lib/*/_build
```

Comment:
I run `mix test --cover` from the `lib/elixir` folder to get a coverage report. Is there a better way?

Also my `mix` binary points to the `mix` binary in the project `bin`.